airqo-platform/AirQo-frontend
# Forgot password

- What does this PR do?
This PR enables a user to update their password after they forgot it - latest implementation. Additionally, more style files for material UI items have been added for quick reference whenever needed.
- What JIRA tasks are related to this PR?
1. [Request access](https://airqoteam.atlassian.net/browse/LOC-16).
2. [Login](https://airqoteam.atlassian.net/browse/LOC-14)
3. [Signout](https://airqoteam.atlassian.net/browse/LOC-13)
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local` to test out the app.
- Which end points should are ready for testing?

1. There are no end points.
2. As you run this, please note that you are also supposed to run the auth-service locally by:
 `cd AirQ0-api/auth-service` Then run `npm install` to get all the necessary dependencies. Afterwards you run `npm start`
3. The updated backend service can be checked out in this [Join Platform (forgot password) API Pull Request](https://github.com/airqo-platform/AirQo-api/pull/57)
5. Also, please remember to include the following to your **.env** file:
```
PORT=5000
NODE_PATH=src
```
